engine/src/test/java/io/camunda/zeebe/engine/processing/timer/TimerStartEventTest.java

```diff
@@ -1197,10 +1197,18 @@ public void shouldAvoidTimeShifting() {
   public void shouldCreateCronTimer() {
     // when
     final long now = engine.getClock().getCurrentTimeInMillis();
+    final long dueDate =
+        ZonedDateTime.ofInstant(Instant.ofEpochMilli(now), ZoneId.systemDefault())
+            .plusHours(1)
+            .withMinute(0)
+            .withSecond(0)
+            .withNano(0)
+            .toInstant()
+            .toEpochMilli();
     final BpmnModelInstance model =
         Bpmn.createExecutableProcess("process")
             .startEvent("start")
-            .timerWithCycle("*/10 * * * * *")
+            .timerWithCycle("0 0 */1 * * *")
```

Comment:
❓ Is the `*/1` necessary for the hour field? I think it means the same as `*` right? Or am I missing something?

Reply:
> I think it means the same as * right? 

Yes, you're right. 😆